pandas/_libs/tslib.pyx

```diff
@@ -66,6 +66,9 @@ from khash cimport (
     kh_init_int64, kh_int64_t,
     kh_resize_int64, kh_get_int64)
 
+from .tslibs.parsing import parse_datetime_string
+from .tslibs.parsing import DateParseError  # noqa
```

Comment:
is DateParseError actually used anywhere here? it doesn't look like it.

Reply:
No.  Retained it because it is used/imported in many places around pandas (and conceivably downstream).  Will change on request.

Reply:
hmm, ok let's fix that in a followup then.